framework/db/Query.php

```diff
@@ -344,7 +344,7 @@ public function sum($q, $db = null)
             return 0;
         }
 
-        return $this->queryScalar("SUM($q)", $db);
+        return $this->toInt($this->queryScalar("SUM($q)", $db));
```

Comment:
`SUM`, `AVG`, `MIN` and `MAX` are availblr for the float values just as well.
Thier result can not be converted to integer.